src/main/java/com/robertx22/age_of_exile/database/data/exile_effects/EffectTags.java

```diff
@@ -11,13 +14,34 @@ public enum EffectTags {
     negative("Negative"),
     heal_over_time("Heal over Time");
 
-    String name;
+    String tag;
 
     EffectTags(String name) {
-        this.name = name;
+        this.tag = name;
     }
 
+
     public String getLocName() {
-        return name;
+        return tag;
+    }
+
+    @Override
+    public AutoLocGroup locNameGroup() {
+        return AutoLocGroup.Misc;
+    }
+
+    @Override
+    public String locNameLangFileGUID() {
+        return SlashRef.MODID + ".effect_tag." + this.name();
+    }
+
+    @Override
+    public String locNameForLangFile() {
+        return this.tag;
+    }
+
+    @Override
+    public String GUID() {
+        return this.name();
```

Comment:
implement IAutoLocName for EffectTag.java.
I just found out that the effect tags don't work at all now, I'll look into it.